meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx_6.1.bb

```diff
@@ -1,3 +1,5 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/linux-lmp-fslc-imx:"
```

Comment:
Wonder if this is really needed here, IIRC this path should already be here by default (this is a bb and not a bbappend).

Reply:
@ricardosalveti Without this I get
```
ERROR: linux-lmp-fslc-imx-rt-6.1.38+gitAUTOINC+42b3766829_7741431d45-r0 do_fetch: Bitbake Fetcher Error: FetchError('Unable to fetch URL from any source.', 'file://0004-FIO-toup-hwrng-optee-support-generic-crypto.patch')
ERROR: Logfile of failure stored in: /home/cryo/work/foundries/lmp/build-lmp-base/tmp-lmp_base/work/imx8mm_lpddr4_evk-lmp-linux/linux-lmp-fslc-imx-rt/6.1.38+gitAUTOINC+42b3766829_7741431d45-r0/temp/log.do_fetch.891017
```

Reply:
Oh ok, this is because the rt recipe includes this one, then the original filespath will indeed be different.

If building just linux-lmp-fslc-imx it should be fine.

I think it is better to move this to linux-lmp-fslc-imx-rt then (to have a filespath covering both recipe names).

Reply:
But this change is fine as well, just add a comment explaining why and we should be good.

Reply:
@ricardosalveti the problem is there're some patches in /linux-lmp-fslc-imx/ and some - in /linux-lmp-fslc-imx/
and we use linux-lmp-fslc-imx non-rt recipe as an include. So we require both directories to be included and it is better to avoid using PN as these paths should be "absolute".

Reply:
> But this change is fine as well, just add a comment explaining why and we should be good.

@ricardosalveti The comment is added.